active/001-tabular-msa.md

```diff
@@ -0,0 +1,82 @@
+---
+Feature name: tabular-msa
+Start date: 2015-08-14
+RFC PR: <leave empty>
+scikit-bio issue: <leave empty>
+---
+
+# Summary
+
+Replace the current experimental `skbio.alignment.Alignment` class with a
+`TabularMSA` class, providing an API for manipulating tabular (row-column)
```

Comment:
Thinking about this more, `MatrixMSA` may be an ambiguous name because a graph can be represented in various matrix forms (adjacency, incidence, Kirchhoff). Since we're making a distinction between total order / linear MSAs and MSAs represented by graphs (partial order and A-Bruijn) it may be best to avoid using "matrix" in the class name. Also,  "matrix" is used to describe other alignment structures, e.g. alignment scoring matrix.